src/main/java/seedu/address/ui/DetailPanel.java

```diff
@@ -105,6 +111,40 @@ public void setPerson(Person person, int index) {
         updateDetails();
     }
 
+    /**
+     * Called when the udders list is cleared.
+     * This method updates the Detail Panel to reflect that no individuals are currently selected.
+     * This is done by invoking {@code updateDetailsToDefault}.
+     */
+    @Override
+    public void uddersCleared() {
+        updateDetailsToDefault();
+    }
+
+    /**
+     * Resets the display details to a default state where no individual is displayed.
+     */
+    private void updateDetailsToDefault() {
+        this.person = null;
+        updateDetails();
+    }
+
+    /**
+     * Clears all person details displayed in the UI.
+     */
+    private void clearDetails() {
+        name.setText("");
+        id.setText("");
+        phone.setText("");
+        address.setText("");
+        email.setText("");
+        role.setText("");
+        major.setText("");
+        tagStart.setText("");
+        tagDetails.getChildren().clear();
+        meetings.setText("");
+    }
```

Comment:
Interesting implementation! Perhaps we can input a kind of instruction instead of a blank space?